src/fsrs/fsrs.ts

```diff
@@ -204,16 +204,13 @@ export class FSRS extends FSRSAlgorithm {
    */
   get_retrievability<T extends boolean>(
     card: CardInput | Card,
-    now: DateInput,
+    now?: DateInput,
     format: T = true as T
-  ): undefined | (T extends true ? string : number) {
+  ): (T extends true ? string : number) {
     const processedCard = TypeConvert.card(card)
-    now = TypeConvert.time(now)
-    if (processedCard.state !== State.Review) {
-      return undefined
-    }
-    const t = Math.max(now.diff(processedCard.last_review as Date, 'days'), 0)
-    const r = this.forgetting_curve(t, +processedCard.stability.toFixed(2))
+    now = now ? TypeConvert.time(now) : new Date()
+    const t = processedCard.state !== State.New ? Math.max(now.diff(processedCard.last_review as Date, 'days'), 0) : 0
+    const r = processedCard.state !== State.New ? this.forgetting_curve(t, +processedCard.stability.toFixed(2)) : 0
```

Comment:
Looping "Again" eight times in the short-term will result in a return of `NaN`.
![image](https://github.com/user-attachments/assets/560051bd-2d99-4c46-b3dc-ac7a775f5e61)

I recommend changing `toFixed(8)` to `8`, ensuring accuracy up to \($10^3$\) without errors.
```suggestion
    const r = processedCard.state !== State.New ? this.forgetting_curve(t, +processedCard.stability.toFixed(8)) : 0
```